Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -87,18 +77,17 @@ public MixedRealityCameraProfile CameraProfile
         }
 
         [SerializeField]
-        [Tooltip("Enable the Input System on Startup")]
+        [Tooltip("Enable the Input System on Startup.")]
         private bool enableInputSystem = false;
 
         /// <summary>
         /// Enable and configure the Input System component for the Mixed Reality Toolkit
         /// </summary>
-        public bool EnableInputSystem
+        public bool IsInputSystemEnabled
         {
             get
             {
-                return inputActionsProfile != null &&
-                       enableInputSystem;
+                return inputActionsProfile != null && enableInputSystem;
```

Comment:
I think we need to add `inputSystemType.Type != null` here.
Otherwise: 
![image](https://user-images.githubusercontent.com/3580640/43484936-a2f8e64e-94c4-11e8-88c0-8fe74e709cb6.png)

Matches line 183 and the boundary manager checks.

Reply:
We should update them all then. Cause the boundary might be the only done doing it correctly

Reply:
Oh we only check the type for two

Reply:
Yeah, it looks like those are the only two with `SystemTypes`